contrib/soju.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=soju IRC bouncer service
+Documentation=https://soju.im/
+Documentation=man:soju(1) man:sojuctl(1)
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+Type=simple
+User=soju
+Group=soju
+DynamicUser=yes
+StateDirectory=soju
+LogsDirectory=soju
+ConfigurationDirectory=soju
+RuntimeDirectory=soju
+ExecStart=/usr/bin/soju --config /etc/soju/config
```

Comment:
The `-config` was required in the past, but is no longer necessary.

Reply:
ah right, it's the default now. should I remove it then?

Reply:
Yes please!